src/apps/DIFFUSION3DPA-Cuda.cpp

```diff
@@ -151,26 +151,24 @@ void DIFFUSION3DPA::runCudaVariant(VariantID vid) {
     DIFFUSION3DPA_DATA_SETUP_CUDA;
 
     using launch_policy =
-        RAJA::expt::LaunchPolicy<RAJA::expt::seq_launch_t,
-                                 RAJA::expt::cuda_launch_t<true>>;
+        RAJA::expt::LaunchPolicy<RAJA::expt::cuda_launch_t<true>>;
```

Comment:
Is the bool template arg used to indicate async launch? Please use a local variable with a descriptive name; e.g., constexpr bool async = true; and pass that so the code is clear.